.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: pymongo-auth-aws
+
+on: [push]
```

Comment:
Does this need pull_request too?:
```
# Triggers the workflow on push or pull request events
on: [push, pull_request]
```
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows